PowerShellTools/DebugEngine/ScriptDebugProcess.cs

```diff
@@ -130,5 +158,16 @@ public int GetPort(out IDebugPort2 ppPort)
             ppPort = _port;
             return VSConstants.S_OK;
         }
+
+        public int QueryCanSafelyAttach()

```

Comment:
Per convo, I this is called by VS during the attach process to give us a chance to warn a user if their connection could be unsafe.